sysutils/dmidecode/src/opnsense/service/conf/actions.d/actions_dmidecode.conf

```diff
@@ -0,0 +1,9 @@
+[system]
+command:/usr/local/sbin/dmidecode -q -t system | grep ':' | tr -d '\t' | awk -F: '{print $1" ="$2}'
+parameters:
+type:script_output
+
+[bios]
+command:/usr/local/sbin/dmidecode -q -t bios | grep ':' | tr -d '\t' | awk -F: '{print $1" ="$2}'
```

Comment:
this one creates a JSON object but the tool is currently not available on OPNsense if you pipe your result through it:
```sh
 jq --slurp --raw-input 'split("\n") | map(split(":") | { (.[0]) : (.[1] | ltrimstr(" "))}) | reduce .[] as $line ({}; . + $line)' 
```

Not a must have but it would be good to always try to return the same format.